crates/oxc_wasm/src/lib.rs

```diff
@@ -170,16 +170,17 @@ impl Oxc {
         let ret = Parser::new(&allocator, source_text, source_type)
             .allow_return_outside_function(parser_options.allow_return_outside_function)
             .parse();
+        let trivias = Rc::new(ret.trivias);
 
-        self.comments = self.map_comments(&ret.trivias);
+        self.comments = self.map_comments(&trivias);
```

Comment:
I think you can leave this as `&ret.trivias` (`map_comments` only needs to temporarily borrow `trivias`). Then wrap `trivias` in an `Rc` further down:

```rs
let semantic_ret = SemanticBuilder::new(source_text, source_type)
    .with_trivias(Rc::new(&ret.trivias))
```

That'd remove one `Rc::clone`.

Reply:
Yes!